test/components/classifiers/test_zero_shot_document_classifier.py

```diff
@@ -0,0 +1,137 @@
+# SPDX-FileCopyrightText: 2022-present deepset GmbH <info@deepset.ai>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+
+from unittest.mock import patch
+
+from haystack import Document
+from haystack.components.classifiers import TransformersZeroShotDocumentClassifier
+from haystack.utils import ComponentDevice, Secret
+
+
+class TestTransformersZeroShotDocumentClassifier:
```

Comment:
Let's add a couple of tests for the init method as well (happy path and otherwise).